code-challenges.js

```diff
@@ -3,39 +3,102 @@
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// function fib() {
+// //     let array = []
+// //     for (let i=0;i<10;i++) {
+// //     if (i===1 ||i=== 0)
+// //         array.push(1)
+// //     else 
+// //      array.push (array [i-1] + array [i-2])
+    
+// // }
+// //         return array
+// // //     }
+// // // console.log(fib())
+// // // // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
+// //             //keeps returning undefined, i'm not sure why
+// // var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+// // // // // Expected output: [-9, 9, 7, 199]
+// // // // var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+// // // // // Expected output: [-823, 7, 23]
+
+// // const isOdd = (array) => {
+// //     const filtered = array.filter(function(x) {
+// //         if(typeof(array) === "number" &&  x % 2 === 1 ||  x % 2 === -1 )
+    
+// //             return array.sort();
+// //     //     } 
+// //     })
+
+// // }
+
+// // console.log(isOdd(fullArr1))
```

Comment:
there are quite a few little syntax errors there that are messing you up. I can see the line of logic you were going down here though and you're on the right track